packages/platform/atoms/availability/wrappers/PlatformAvailabilitySettingsWrapper.tsx

```diff
@@ -0,0 +1,102 @@
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+
+import type { ScheduleLabelsType } from "@calcom/features/schedules/components/Schedule";
+import { showToast } from "@calcom/ui";
+
+import { useMe } from "../../hooks/useMe";
+import useClientSchedule from "../hooks/useClientSchedule";
+import useDeleteSchedule from "../hooks/useDeleteSchedule";
+import useUpdateSchedule from "../hooks/useUpdateSchedule";
+import { daysInAWeek } from "../lib/daysInAWeek";
+import { PlatformAvailabilitySettings } from "../settings/index";
+import type { AvailabilityFormValues } from "../types";
+
+type PlatformAvailabilitySettingsWrapperProps = {
+  id?: string;
+  labels?: {
+    tooltips: ScheduleLabelsType;
+  };
+};
+
+const defaultLabels = {
+  tooltips: {
+    addTime: "Add new time slot",
+    copyTime: "Copy times to …",
+    deleteTime: "Delete",
+  },
+} as const;
```

Comment:
all the translations should be here, so that platform customers can change them, can even put it in a separate file, and then use this const as type of component props